### ISSUE TITLE
Feature Request - Show MonoBehaviour script icon for objects that contain scripts

### ISSUE THREAD

**Issue by nankink** [WooshiiDev/HierarchyDecorator#65]
If there's already an option for this feature, excuse my clumsiness! Would appreciate if someone could show me how!

**Feature Description**
Showing MonoBehaviour icon for GameObjects that contain a script, without the need to specify each script in the settings. Simply have an MonoBehaviour derived class and it would show up in the hierarchy. Also, having only 1 icon as in the [other feature request ](https://github.com/WooshiiDev/HierarchyDecorator/issues/49) would be awesome!

**Reason for feature**
Quickly identifying what game object contains scripts (no matter which one) would also be an incredible help!

**Comment by WooshiiDev** [comment]
Hello nankink,
I'm linking to the pull request #67 as someone has went ahead and made this a feature addition.
There are some concerns about the codebase changes I've been working on, but either way this will be coming.

**Comment by nankink** [comment]
> Hello nankink,
> I'm linking to the pull request #67 as someone has went ahead and made this a feature addition.
> There are some concerns about the codebase changes I've been working on, but either way this will be coming.
> 
> Edit: PR is currently closed but will be reopened when required.

This is awesome!! To be honest, wasn't expecting that this would be added to the tool, and this fast. 

I wish I could contribute to the project as well, but my knowledge in editor code is not a lot ://

Thanks a lot!

**Comment by WooshiiDev** [comment]
I've been working on a ton of stuff and will be getting ready to release a new version quite soon. Got some improvements to do but it's getting there.

Alright so I've added a small toggle to show a single MonoBehaviour script on each instance that has one. 
To _extend_ what already existed I've added two new buttons that will allow you to independently toggle on built-in components and custom one automatically.

![HD_Show_Toggles_#2](https://user-images.githubusercontent.com/31889435/220519821-948877d5-e5df-46aa-b1b8-e9809d0aa9db.gif)

How does this look? Did I get anything wrong? Also as @SiarheiPilat originally made a PR for this (I should have pulled it after being quiet for months), think its fair to ask how you feel about this too.

Should be released within the next few weeks, but it's currently on the [develop](https://github.com/WooshiiDev/HierarchyDecorator/tree/develop) branch. Would note that develop might be a little unstable if you upgrade from your current version to it; just in case you try it out.
